src/install.sh

```diff
@@ -55,13 +60,18 @@ if \
 	[ -f "$SPATH/pihole-updatelists.conf" ] && \
 	[ -f "$SPATH/pihole-updatelists.service" ] && \
 	[ -f "$SPATH/pihole-updatelists.timer" ] && \
-	[ -f "$SPATH/pihole-updatelists.bash" ] \
+	[ -f "$SPATH/pihole-updatelists.bash" ] && \
+	([ "$DOCKER_INSTALL" == 0 ] || ([ "$DOCKER_INSTALL" == 1 ] && [ -f "$SPATH/60-pihole-updatelists.sh" ]))  \
 ; then
 	cp -v $SPATH/pihole-updatelists.php /usr/local/sbin/pihole-updatelists && \
 	chmod -v +x /usr/local/sbin/pihole-updatelists
 	
-	if [ ! -f "/etc/pihole-updatelists.conf" ]; then
-		cp -v $SPATH/pihole-updatelists.conf /etc/pihole-updatelists.conf
+	if [ ! -d "/etc/pihole-updatelists" ]; then
+		mkdir /etc/pihole-updatelists
+	fi
+	
+	if [ ! -f "/etc/pihole-updatelists/pihole-updatelists.conf" ]; then
+		cp -v $SPATH/pihole-updatelists.conf /etc/pihole-updatelists/pihole-updatelists.conf
 	fi
 
 	if [ "$SYSTEMD" == 1 ]; then
```

Comment:
Systemd stuff probably shouldn't be installed inside docker as it will be useless? Needs to be excluded by `DOCKER_INSTALL=1`